src/bin/pgcopydb/extensions.c

```diff
@@ -200,31 +200,71 @@ copydb_copy_extensions_hook(void *ctx, SourceExtension *ext)
 		{
 			SourceExtensionConfig *config = &(ext->config.array[i]);
 
-			log_info("COPY extension \"%s\" "
-					 "configuration table \"%s\".\"%s\"",
-					 ext->extname,
-					 config->nspname,
-					 config->relname);
-
 			char qname[PG_NAMEDATALEN_FQ] = { 0 };
 
 			sformat(qname, sizeof(qname), "%s.%s",
 					config->nspname,
 					config->relname);
 
-			CopyArgs args = {
-				.srcQname = qname,
-				.srcAttrList = "*",
-				.srcWhereClause = config->condition,
-				.dstQname = qname,
-				.dstAttrList = "",
-				.bytesTransmitted = 0
-			};
-
-			if (!pg_copy(src, dst, &args))
+			/*
+			 * According to the PostgreSQL documentation, relkind 'r' is a
+			 * regular table, and 'S' is a sequence.
+			 * https://www.postgresql.org/docs/current/catalog-pg-class.html
+			 */
+			if (config->relkind == 'r')
+			{
+				CopyArgs args = {
+					.srcQname = qname,
+					.srcAttrList = "*",
+					.srcWhereClause = config->condition,
+					.dstQname = qname,
+					.dstAttrList = "",
+					.bytesTransmitted = 0
+				};
+
+				log_info("COPY extension \"%s\" "
+						 "configuration table %s",
+						 ext->extname,
+						 qname);
+
+				if (!pg_copy(src, dst, &args))
+				{
+					/* errors have already been logged */
+					return false;
+				}
+			}
+			else if (config->relkind == 'S')
+			{
+				SourceSequence seq = { };
```

Comment:
Is it missing a zero (`seq = { 0 }`) or is that spelling doing the same thing anyway? In order to look like the rest of the code I think we should add the zero in there, but I'm curious of maybe having missed something...

Reply:
Done :)